test_result.txt

```diff
@@ -0,0 +1,65 @@
+============================= test session starts ==============================
```

Comment:
Yes I know we normally don't do this, but I was trying to allow you to see what I was seeing as I was stuck. The hope was that in seeing the error wherever it was I ended up getting stuck you could suggest a remedy. The idea is to put the test in here for now while writing the code and remove it later.